utilities/backupable/backupable_db.cc

```diff
@@ -1281,14 +1280,25 @@ Status BackupEngineImpl::DeleteBackupInternal(BackupID backup_id) {
   return Status::OK();
 }
 
-void BackupEngineImpl::GetBackupInfo(std::vector<BackupInfo>* backup_info) {
+void BackupEngineImpl::GetBackupInfo(std::vector<BackupInfo>* backup_info,
+                                     bool include_file_details) {
   assert(initialized_);
   backup_info->reserve(backups_.size());
   for (auto& backup : backups_) {
-    if (!backup.second->Empty()) {
-      backup_info->push_back(BackupInfo(
-          backup.first, backup.second->GetTimestamp(), backup.second->GetSize(),
-          backup.second->GetNumberFiles(), backup.second->GetAppMetadata()));
+    const auto& meta = *backup.second;
```

Comment:
I'd rather replace `auto` for clarity. Parens here seem like excessive syntax. I'm getting a `BackupMeta` from each element of `backups_`.

Reply:
Perhaps it is just me being old, but I can never remember the precedence of "*" and ".", and I am not quite sure they have not changed since I started doing C/C++.  And when I see "*foo.", I naturally think it should be "foo->", which is not true here at all.  Hence the request for parens.